src/Magento/FunctionalTestingFramework/Test/Handlers/TestObjectHandler.php

```diff
@@ -110,7 +110,7 @@ public function getTestsByGroup($groupName)
         foreach ($this->tests as $test) {
             /** @var TestObject $test */
             if (in_array($groupName, $test->getAnnotationByName('group'))) {
-                $relevantTests[$test->getName()] = $test;
+                $relevantTests[$test->getName()] = $this->extendTest($test);
```

Comment:
Is this something we can add a verification test for?

Reply:
Possibly. I will look into it.